src/main/resources/jupyter/init-actions.sh

```diff
@@ -334,4 +334,11 @@ make install
 ldconfig
 python3 --version
 
+export GCSFUSE_REPO=gcsfuse-`lsb_release -c -s`
+echo "deb http://packages.cloud.google.com/apt $GCSFUSE_REPO main" | sudo tee /etc/apt/sources.list.d/gcsfuse.list
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+
+sudo apt-get update
+sudo apt-get install -y -q gcsfuse
+
```

Comment:
It would be good to do the `gcsfuse` install after the following log message, and maybe surround with `log` messages for better tracking.